server/tests/src/test/java/org/infinispan/server/functional/rest/RestProtobufResource.java

```diff
@@ -0,0 +1,42 @@
+package org.infinispan.server.functional.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.infinispan.test.TestingUtil.join;
+
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.client.rest.configuration.Protocol;
+import org.infinispan.client.rest.configuration.RestClientConfigurationBuilder;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.commons.util.Util;
+import org.infinispan.rest.assertion.ResponseAssertion;
+import org.infinispan.server.functional.ClusteredIT;
+import org.infinispan.server.test.api.TestClientDriver;
+import org.infinispan.server.test.junit5.InfinispanServer;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+public class RestProtobufResource {
```

Comment:
@jabolina could we rename this file, please ? makes think is an actual class instead of a test

Reply:
:+1: to naming it *Test as we are doing that actually https://github.com/infinispan/infinispan/pull/15231/files#diff-b7bb2105a3c3bfe5ab29dd75bb03f4c925bb7cdc5dd3d1ad5b69190701ff8224R73

Reply:
I've checked: the `server/tests` pom is disabling surefire, we should be able to safely use `*Test`, although it will only be executed as part of a parent `IT` test from failsafe